cafebabel/static/js/script.js

```diff
@@ -113,3 +113,13 @@ Array.from(document.querySelectorAll('h1.edit input')).forEach(inputh1 => {
     () => inputh1.value || h1.classList.remove('active')
   )
 })
+
+/* animation picto sections */
+Array.from(document.querySelectorAll('.svg-animation')).forEach(svg => {
+  window.addEventListener('scroll', function() {
+    if (svg.getBoundingClientRect().top < 600) {
```

Comment:
Maybe document why `600`?

Reply:
Yea I didn't know how to get the moment where the picto start being watched. Maybe `viewportHeight` will do https://github.com/cafebabel/cafebabel.com/pull/331/commits/d457cd071a42183c58ade31af23c4cbca1949e7c